Update raw-data-export-event-hub.md (#9097)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-atp/raw-data-export-storage.md

## Enable raw data streaming:

> 1. Log in to [Microsoft Defender for Endpoint portal](https://securitycenter.windows.com) with ***Global Administrator*** or ***Security Administrator*** user.

Reviewer: ```suggestion
1. Log in to [Microsoft Defender for Endpoint portal](https://securitycenter.windows.com) as a ***Global Administrator*** or ***Security Administrator***.
```

> 2. Go to [Data export settings page](https://securitycenter.windows.com/interoperability/dataexport) on Microsoft Defender Security Center.

Reviewer: ```suggestion
2. Go to the [Data export settings page](https://securitycenter.windows.com/interoperability/dataexport) on Microsoft Defender Security Center.
```
